locksmith/migrations/20211202121847-create-lock-migrations.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('LockMigrations', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      lockAddress: {
+        type: Sequelize.STRING,
+      },
+      newLockAddress: {
+        type: Sequelize.STRING,
+      },
+      chain: {
```

Comment:
Bon je me dis que ca pourrait etre cool plus tard de migrer d'une chaine a l'autre? On rajoute un `newLockChain` ?

Reply:
let's wait to have an implementation to add a migration? With sequelize its quite straightforward